library/templates/dcp_developments.yml

```diff
@@ -0,0 +1,56 @@
+dataset:
+  name: &name dcp_developments
+  version: "{{ version }}"
+  acl: public-read
+  source:
+    url:
+      path: https://nyc3.digitaloceanspaces.com/edm-publishing/db-developments/main/{{ version }}/output/devdb.csv
```

Comment:
What was the behavior of devdb when you ingested the shapefile? Did it just not work?

Reply:
It did work, but output had slightly different column names than expected (lack of underscore in one column at least)